internal/pkg/remote/remote_test.go

```diff
@@ -72,6 +88,39 @@ func TestWriteToReadFrom(t *testing.T) {
 		})
 	}
 
+	for _, test := range testsFail {
+		t.Run(test.name, func(t *testing.T) {
+			yaml, err := yaml.Marshal(test.data)
+			if err != nil {
+				t.Fatalf("cannot mashal YAML: %s\n", err)
+			}
+
+			// We manually create the file to test ReadFrom()
+			tempFile, err := ioutil.TempFile("", "")
```

Comment:
I think its cleaner to implement this test using a `bytes.Buffer` like `testsPass` is since both types implement the same interfaces

Reply:
Good point and done.